README.md

```diff
@@ -71,6 +71,15 @@ consumers. They also take effect when searching events or streaming
 them to clients. If multiple filters are chosen they are combined
 using logical AND condition. The following filters are available:
 
+* `exceptions`
+* `level`
+* `pattern`
+* `start-time`
+* `end-time`
+* `threads`
+* `loggers-allowlist`
```

Comment:
Let's use `:logger-allow-list` and `:logger-block-list`? Having a hyphen in between reads better (in my opinion) and does not trip the spell checker (in this Github form). Also loggers -> logger? But feel free to ignore this suggestion if you don't like it.

Reply:
Thanks for the suggestions. I've seen `allowlist` quite often so I'd assume a good chunk of our audience has, as well.

I'm ambivalent about logger vs. loggers. Maybe 'loggers' is a touch more clear (as it reminds Emacs users that it's a list of things - _plural_)